### PR TITLE
Fs aug19 slava

### PR DESCRIPTION
**Pull request by MMashivskij**

### CODE REVIEW

File: src/transformState.js [hunk 1]
```diff
@@ -49,7 +49,23 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  transforms.forEach(function(item, i, arr) {
```
Comment (gestap) [inline src/transformState.js]: why you need `arr` and `i`?
use arrow function for method
```suggestion
  transforms.forEach((item) => {
```

File: src/transformState.js [hunk 2]
```diff
@@ -49,7 +49,23 @@
...
+    if (item.operation === 'addProperties') {
+      for (const key in item.properties) {
+        state[key] = item.properties[key];
```
Comment (gestap) [inline src/transformState.js]: better to use `Object.assign` for this task

File: src/transformState.js [hunk 3]
```diff
@@ -49,7 +49,23 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  transforms.forEach(function(item, i, arr) {
+    if (item.operation === 'addProperties') {
```
[the remark this answers is not in the document]
Comment (gestap) [reply src/transformState.js]: also, try to use destruction method

File: src/transformState.js [hunk 4]
```diff
@@ -49,7 +49,28 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  transforms.forEach(item => {
```
Comment (gestap) [inline src/transformState.js]:
```suggestion
  transforms.forEach((item) => {
  const [operation, properties] = item;
```

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**